pkg/udev/mockdata.go

```diff
@@ -151,7 +154,9 @@ func getSyspathOfOsDisk(osDiskName string) (string, error) {
 
 // getOsDiskSize returns size of os disk in success
 func getOsDiskSize(osDiskName string) (string, error) {
-	sizeByte, err := ioutil.ReadFile(filepath.Clean("/sys/class/block/" + osDiskName + "/size"))
+	basePath := "/sys/class/block/"
+	fileName := "size"
+	sizeByte, err := ioutil.ReadFile(filepath.Clean(path.Join(basePath, osDiskName, fileName)))
```

Comment:
```suggestion
	sizeByte, err := ioutil.ReadFile(filepath.Clean(path.Join("/sys/class/block/", osDiskName, "size")))
```

We don't seem to be using the variables `fileName` and `basePath` anywhere else in the function. I think we can directly use the strings  for one time use.